tests/unit/model/test_datastore.py

```diff
@@ -442,3 +429,25 @@ def test_failed_bump(
         bump_manifesto = DatastoreVersion(**json.load(f))
     datastore.bump_version(JOB_ID, bump_manifesto, "description")
     datastore.refresh.assert_called_once()
+
+
+def test_rollback_of_remove_operation(requests_mock: RequestsMocker):
+    requests_mock.put(
+        f"{JOB_SERVICE_URL}/jobs/{JOB_ID}", json={"message": "OK"}
+    )
+    DATASET_NAME = "FOEDSELSVEKT"
+    DESCRIPTION = "Setter til remove"
+
+    datastore.remove(JOB_ID, DATASET_NAME, DESCRIPTION)
+    datastore.delete_draft(JOB_ID, DATASET_NAME, rollback_remove=True)
+    assert len(requests_mock.request_history) == 4
+    with open(DRAFT_VERSION, encoding="utf-8") as f:
+        draft_version = json.load(f)
+    assert not os.path.exists(draft_data_path(DATASET_NAME))
+    assert not os.path.exists(partitioned_draft_data_path(DATASET_NAME))
```

Comment:
We don't need these checks. There is never draft data present before or after our changes.
```suggestion
```